data/completion/snap

```diff
@@ -32,8 +32,14 @@ _complete() {
         try)
             _filedir -d
             ;;
+        connect)
```

Comment:
Personally I have no idea what just happened. It would be good to comment on what is being done below.